server/camphoric/models.py

```diff
@@ -57,7 +57,11 @@ class Event(TimeStampedModel):
     camper_schema = JSONField(null=True)
     payment_schema = JSONField(null=True)
     registration_schema = JSONField(null=True)
-    lodging_schema = JSONField(null=True)
+    registration_ui_schema = JSONField(null=True)
```

Comment:
I think these fields should probably have some comments describing what they mean semantically -- it's not clear what each of these schemas represent and why they differ from one another.

Reply:
Good thought - done.